CryptoBinPkg/Driver/Packaging/generate_cryptodriver.py

```diff
@@ -599,7 +599,7 @@ def get_crypto_dsc(options, functions):
         lines.append(f"  DEFINE RUNTIMEDXE_CRYPTO_DRIVER_FILE_GUID = {runtime_dxe_guid}")
         lines.append(f"  DEFINE SMM_CRYPTO_DRIVER_FILE_GUID = {smm_guid}")
         lines.append(f"  DEFINE STANDALONEMM_CRYPTO_DRIVER_FILE_GUID = {standalone_mm_guid}")
-        lines.append(f"!endif\n")
+        lines.append("!endif\n")

```

Comment:
Just curious why this change was necessary?  Did it cause a build error?

Reply:
I have a Ruff extension installed in VScode. It made the change automatically when I saved the file. 

I will revert its change.